### PR TITLE
Add defcustom for bash-completion-args

### PR DESCRIPTION
**Pull request by svend**
This allows passing custom flags, such as `--login`, to BASH.

### CODE REVIEW

**Comment by svend** [comment]
On my system, bash_completion is loaded in my `.bash_profile`, so it only gets loaded for login shells. I need a way to add the `--login` flag to bash to get completions to work in Emacs.

File: bash-completion.el [hunk 1]
```diff
@@ -874,11 +879,12 @@ is set to t."
...
-		   "*bash-completion*"
-		   (generate-new-buffer-name " bash-completion")
-		   bash-completion-prog
-		   "--noediting"))
+		  (apply 'start-process
+                         (append
+                          `("*bash-completion*"
+                            ,(generate-new-buffer-name " bash-completion")
+                            ,bash-completion-prog)
+                          bash-completion-args))
```
Comment (szermatt) [inline bash-completion.el]: A parentheses is missing to close the (set

**Comment by svend** [comment]
Sorry about that. I've fixed the syntax and pushed a new commit.

**Comment by szermatt** [comment]
Thank you!

**Pull request merged**